### PR TITLE
docs(rxFavicon): FRMW-174 Migrate rxFavicon

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-174

- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/rxFavicon/rxFavicon.js [hunk 1]
```diff
@@ -16,14 +16,18 @@ angular.module('encore.ui.rxFavicon', ['encore.ui.rxEnvironment'])
...
+ * This is a congifuration object for each environment.
+ * @param {String=} rxFavicon.staging This defines the favicon file for the staging environment.
+ * @param {String=} rxFavicon.local This defines the favicon file for the local environment.
  * @example
  * <pre>
- * TBD
+ * <link rel="icon"
+    type="image/png"
+    href="favicon.png"
+    rx-favicon="{ staging: 'staging-favicon.png', local: 'local-favicon.png' }" />
```
Comment (bry4n) [inline src/rxFavicon/rxFavicon.js]: Oops. you are missing three asterisks

```js
/**
* <pre>
* <link rel="icon"
*     type="image/png"
*     href="favicon.png"
*     rx-favicon="{ staging: 'staging-favicon.png', local: 'local-favicon.png' }" />
* </pre>
*/
```

**Comment by 100stacks** [comment]
Hey looks like you might have missed one file...this should be removed from `the src/rxFavicon/README.md`, since you captured it in `src/rxFavicon/rxFavicon.js`
```
## Usage

rxFavicon is an attribute based directive. It accepts an object with the following optional parameters defined:

 - **staging**: Path to favicon to use for staging environments (falls back to 'production' icon)
 - **local**: Path to favicon to use for staging environments (falls back to 'staging' icon)

See the demo examples for code examples.
```

**Comment by 100stacks** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

@ssalinas24, please squash commits

**Pull request merged**